### PR TITLE
Add in bash completion

### PR DESCRIPTION
**Pull request by cwayne18**
Add in super simple bash completion

### CODE REVIEW

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

**Comment by kyrofa** [comment]
Nice, thanks Chris! Something I've been wanting myself. However, could you please log a bug on this issue and mention it in the commit (as discussed in the [contribution guide](https://github.com/ubuntu-core/snapcraft/blob/master/CONTRIBUTING.md)?

**Comment by cwayne18** [comment]
Done and done :)

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

File: debian/control [hunk 1]
```diff
@@ -24,6 +24,7 @@ Build-Depends: debhelper (>= 9),
                python3-testscenarios,
                python3-xdg,
                python3-yaml,
+               bash-completion,
```
Comment (kyrofa) [inline debian/control]: Please keep this list sorted alphabetically.

**Comment by kyrofa** [comment]
Just tested, this works great :+1: .

**Pull request merged**